spark/src/test/java/com/netflix/iceberg/spark/data/RandomData.java

```diff
@@ -103,7 +113,7 @@ public Object map(Types.MapType map, Supplier<Object> valueResult) {
 
       Map<String, Object> result = Maps.newLinkedHashMap();
       for (int i = 0; i < numEntries; i += 1) {
-        String key = randomString(random) + i; // add i to ensure no collisions
+        String key = randomString(random).toString() + i; // add i to ensure no collisions
```

Comment:
Why would `randomString` return something that isn't a String?

Reply:
randomString returns a UTF8String.

Reply:
Ah, makes sense. Thanks!